code/modules/mob/living/carbon/human/human.dm

```diff
@@ -115,7 +115,7 @@
 			. += "Primary Objective: [html_decode(assigned_squad.primary_objective)]"
 		if(assigned_squad.secondary_objective)
 			. += "Secondary Objective: [html_decode(assigned_squad.secondary_objective)]"
-	if(faction == FACTION_MARINE)
+	if(faction == FACTION_MARINE || (FACTION_MARINE in wear_id.faction_group))
```

Comment:
```suggestion
	if(faction == FACTION_MARINE || (FACTION_MARINE in wear_id?.faction_group))
```
Ever you take off your id, this runtimes: 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/ad257792-8d6a-44b3-87fa-8d1fbdf021f4)